pyblish/plugin.py

```diff
@@ -1326,6 +1390,7 @@ def discover(type=None, regex=None, paths=None):
 
         plugins[plugin.__name__] = plugin
 
+    filter_plugins(plugins)
```

Comment:
I think we should move this till the very last step; after conversion to a list, and after having been sorted. That would give the user final control over the items, including their order.